packages/webpack-config-anansi/media.d.ts

```diff
@@ -0,0 +1,72 @@
+declare module "*.svg" {
+  let ReactComponent: import('react').ComponentType<any>;
```

Comment:
Here if we want, we can be as specific as this:
`ReactComponent: React.SFC<React.SVGProps<SVGSVGElement>>;`

Reply:
nice!